packages/contracts-bedrock/scripts/Artifacts.s.sol

```diff
@@ -231,27 +219,7 @@ abstract contract Artifacts {
 
     /// @notice Adds a deployment to the temp deployments file
     function _appendDeployment(string memory _name, address _deployed) internal {
```

Comment:
IMO it would be much cleaner to remove this function which is called only once, and replace it with the call to vm.writeJSON. 

I will approve for now and let you resolve how you see fit.